llvm/include/llvm/ProfileData/InstrProfReader.h

```diff
@@ -397,8 +387,8 @@ class RawInstrProfReader : public InstrProfReader {
             Correlator)),
         BIDFetcher(BIDFetcher),
```

Comment:
do we need merge
```
BIDFetcher(BIDFetcher),
BIDFetcherCorrelatorKind(BIDFetcherCorrelatorKind), Warn(Warn) {}
```
into SYCLomatic_CUSTOMIZATION part?

Reply:
Done